index.html

```diff
@@ -15,6 +15,7 @@
                 <div id="header-placeholder"></div>
                 <ul id="tab-container">
                     <li id="new-tab" class="tab tab-new" onclick="onNewTab()"><i class="fa fa-plus tab-new-icon"></i></li>
+                    <li id="new-tab" class="tab tab-new" onclick="onNewPrivateTab()" title="Private/Incognito mode"><i class="fa fa-plus tab-new-private-icon"></i></li>
```

Comment:
Let's have onNewTab take a boolean argument and make both call onNewTab instead.